.github/workflows/build.yml

```diff
@@ -0,0 +1,146 @@
+name: Prebuild
+
+on:
+  push:
+    # tags:
+    #   - v*
+
+env:
+  NAPI_VERSION: 3
+  BINDING_NAME: HID
+  BINDING_NAME_RAW: HID_hidraw
+
+jobs:
+  build-and-test:
+    name: Build ${{ matrix.arch }} on ${{ matrix.os }} ${{ matrix.libc }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # windows
+          - os: windows-2019
+            arch: x64
+            is-native: true
+          - os: windows-2019
+            arch: ia32
+            is-native: false
+          # macos
+          - os: macos-11
+            arch: arm64
+            is-native: false
+          - os: macos-10.15
+            arch: x64
+            is-native: true
+          # linux
+          - os: ubuntu-18.04
+            arch: x64
+            is-native: true
+          - os: ubuntu-latest
+            arch: arm64
+            is-native: false
+            docker-arch: linux/arm64
+            docker-image: node:14-buster
+          - os: ubuntu-latest
+            arch: arm
+            is-native: false
+            docker-arch: linux/arm/v7
+            docker-image: node:14-buster
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Checkout submodules
+        shell: bash
+        run: |
+          auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+          git submodule sync --recursive
+          git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+
+      - name: Install Linux dependencies
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libudev-dev libusb-1.0-0-dev
+
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
```

Comment:
one difference is that the tests are only going to be run against this one version of node.
That shouldn't be an issue because node-api means that the binary is compatible with a wide range of versions, but there might be value in running the tests against multiple to verify that.
@todbot thoughts?